src/style.css

```diff
@@ -0,0 +1,220 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  height: 100vh;
+  background-color: #f1f1f1;
+}
+
+.d-flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.d-flex-sb {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.todo-container {
+  max-width: 500px;
+  width: 100%;
+  margin: auto auto;
+  border-radius: 3px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.25);
+  padding-bottom: 1em;
+  background-color: #f1f1f1;
+  color: #545862;
+}
+
+li,
+.todo-list-title {
+  padding: 1em;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+  background-color: white;
+}
+
+.todo-list-title .fa-sync {
+  font-size: 14px;
+}
+
+li:nth-of-type(2) {
+  padding: 1em 1em;
+}
+
+.todo-list-title {
+  border-top-left-radius: 3px;
+  border-top-right-radius: 3px;
+}
+
+.todo-list-title i {
+  color: rgba(0, 0, 0, 0.3);
+}
+
+.todo-list-title i:hover {
+  cursor: pointer;
+  color: rgba(0, 0, 0, 0.6);
+}
+
+.add-task-container {
+  padding: 0 1em;
+}
+
+.add-task-form {
+  width: 100%;
+}
+
+.add-task-form input,
+.add-task-form input:hover,
+.add-task-form input:focus {
+  border: 0;
+  outline: 0;
+  color: #545862;
+}
+
+.add-task-form .new-task {
+  flex-grow: 1;
+  padding: 1em 0;
+}
+
+.add-task-form .new-task::placeholder {
+  color: rgba(0, 0, 0, 0.4);
+}
+
+.add-task-form .input-return {
+  background-image: url(https://web.archive.org/web/20180728085226im_/http://www.getminimalist.com/assets/enter-b06ec7fb441bc0e40d61ba975a0291ad.png);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-color: transparent;
+  background-size: 10px;
```

Comment:
Here is a good place to use the background shorthand, according to the KISS pprinciple

Reply:
I like this suggestion but I struggle with that since I keep forgetting what comes before what, maybe you could help me with a tool or something that eases things?
@Lino09